### PR TITLE
MQE-987: Decouple MFTF from Magento

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:
- [ ] New topic
- [x] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
 
When this pull request is merged, it will...
 

## Additional information

whatsnew
MFTF 2.3: The `robo` CLI has been moved to [`mftf'](https://devdocs.magento.com/guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.html).

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2052) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2052) before we can accept your contribution.<br/>

**Comment by dshevtsov** [comment]
@imeron2433 will these changes be applicable for Magento 2.3 only?

File: guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+##### Arguments:
+*arg*                | description|
+|---|---|
+`name` | name(s) of specific tests to generate
+
+##### Options:
+opt                | description|
+|---|---|
+`--config=CONFIG`   | default, singleRun, or parallel [default: "default"]  
+`--force`           | force generation of tests regardless of Magento Instance Configuration
```
Comment (dshevtsov) [inline guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: It is not clear what means *regardless of Magento Instance Configuration*

Comment (dshevtsov) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: What configuration parameters or files are ignored?

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: We ask Magento for the module read order when we parse the modules in, this is a flag to ignore the merge order and just parse the entire filesystem. Make sense?

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: not sure I understand what you mean with your second comment?

Comment (dshevtsov) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: __regardless__ is the same as __ignoring__
Just wanted to understand what exactly that flag forces to ignore.
Your explanation is clear, thank you.

File: guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+
+{% include note.html
+type="info"
+content="The strings must be escaped and surrounded in quotes."
+%}
+
+#### `generate:suite`
+
+##### Description:
+Generates a single suite based on declaration in xml
```
Comment (dshevtsov) [inline guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: There is a collision.
The __Description__ tells about the _single_ suite, but the __Usage__ uses _multiple_ suites.

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: Ahh good point, this actually allows for multiple suites to be generated depending on how many args are passed in, I will fix it.

File: guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+##### Arguments:
+arg                | description|
+|---|---|
+  suites | suite names for generation (separated by space)
+
+### run
+#### `run:group`
+
+##### Usage:
+  `run:group [options] [--] <groups> (<groups>)...`
```
Comment (dshevtsov) [inline guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: What does the `--` flag mean?
How the groups must be listed in the command?
Add an example of the command with multiple groups.

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: bin/mftf run:group -k -- group1 group2 group3

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: the "--" separates the options from the arguments

Comment (dshevtsov) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: should it start from `vendor/bin/mftf` like in previous examples?

Comment (dshevtsov) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: `run:group group1 group2 group3`
Is it correct example of the command with no options?

Comment (imeron2433) [reply guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md]: yes and yes

**Comment by dshevtsov** [comment]
@imeron2433 
I updated the PR, please review the changes.

File: guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 4]
```diff
@@ -145,23 +147,26 @@ content='The strings must be escaped and surrounded in quotes.'
...
-Generates a single suite based on declaration in xml.
+Generates suite(s) based on XML declarations.
 
 #### Usage
 
-`generate:suite <suites> (<suites>)...`
+`generate:suite <suites> <suites> ...`
 
-#### Arguments
+`<suites>` A suite names for generation (separated by space)
```
Comment (imeron2433) [inline guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md]: I think this should say either 'the suite names' or 'suite names'

**Pull request merged**